client/src/components/Forms/signup/Form.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Form,
+  Input,
+  message,
+} from 'antd';
+import { useDispatch } from 'react-redux';
+import './signup.css';
+import axios from 'axios';
+import { signUpAction } from '../../../redux/features/auth/authSlice';
+
+const confirmPasswordHandel = (getFieldValue) => ({
+  validator(_, value) {
+    if (!value || getFieldValue('Password') === value) {
+      return Promise.resolve();
+    }
+    return Promise.reject(new Error('The two passwords that you entered do not match!'));
+  },
+});
+
+const SignupForm = () => {
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirm, setConfirm] = useState('');
+  const dispatch = useDispatch();
+
+  const signUp = async () => {
+    try {
+      const { data } = await axios.post('/api/signup', {
+        fullName,
+        email,
+        password,
+        confirm,
+        role: 'interviewee',
+      });
+      dispatch(signUpAction(data));
+      message.success(`Welcome ${fullName}, Signup success`);
```

Comment:
Receive the message from the back and show it, so we tell the users that they need to verify their accounts.